remote-wallet/src/ledger.rs

```diff
@@ -424,7 +428,7 @@ pub fn get_ledger_from_info(
     info: RemoteWalletInfo,
     keypair_name: &str,
     wallet_manager: &RemoteWalletManager,
-) -> Result<Arc<LedgerWallet>, RemoteWalletError> {
+) -> Result<Arc<Mutex<LedgerWallet>>, RemoteWalletError> {
```

Comment:
Seems weird that `Arc` is mentioned in the `ledger.rs` module. It doesn't do multi-threading, right? It's users do?  Weirder that a `Mutex` got pulled in too.